src/gvmd.c

```diff
@@ -2325,9 +2325,7 @@ gvmd (int argc, char** argv, char *env[])
 #if CVSS3_RATINGS == 1
       printf ("CVSS3 severity ratings enabled\n");
 #endif
-#if COMPLIANCE_REPORTS == 1
       printf ("Compliance reports enabled\n");
```

Comment:
I think this can also be removed since compliance reports will always be enabled.